guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -10,13 +10,21 @@ This information allows Katello to display manifest labels, annotations, and inf
 As a result, migrations must be performed to pull this content from manifests into the database.
 
 ifdef::katello[]
-This migration takes time, so if you depend on container content and need minimal upgrade downtime, run the following command on {ProjectServer} for a pre-migration:
+.Procedure
+. This migration takes time, so if you depend on container content and need minimal upgrade downtime, enter the following commands in a `tmux` window on {ProjectServer} for a pre-migration:
++
+[options="nowrap" subs="attributes"]
+----
+# {foreman-maintain} advanced procedure run pulpcore-container-handle-image-metadata
+----
+This command migrates data while {Project} is running without any need for downtime and reduces future upgrade downtime.
 
+. To allow the container image API to display manifest labels, annotations, and if the manifest represents bootable or flatpak content, enter the following command:
++
 [options="nowrap" subs="attributes"]
 ----
-# foreman-maintain advanced procedure run pulpcore-container-handle-image-metadata
+# foreman-rake katello:import_container_manifest_labels
```

Comment:
It would be good to describe the commands in a consistent way. Right now, in the first step, you start with the command and follow with a description of what the command does; in the second step, you do things the other way. (Looking at the surrounding procedures, they start with a description.)